jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -2705,6 +2705,397 @@ YUI.add('juju-env-api', function(Y) {
     },
 
     /**
+      Return information about all entities in this model.
+
+      @param {Function} callback A callable that must be called once the
+        operation is performed. It will receive two arguments: an error string
+        and an empty object if an error occurred, null and the status result
+        otherwise. The status result is an object similar to the following:
+
+          {
```

Comment:
This can be a great resource for anyone implementing this but will this ever change? Is it missing some things or is this everything possible from status? When it gets out of date will it still be helpful or just filling up lines in the file? Maybe a better thing would be a command on how to generate this output?